utils/centos7-ci.sh

```diff
@@ -57,11 +57,11 @@ install_dependencies() {
     # install dependencies
     git clone https://github.com/iresty/test-nginx.git test-nginx
     make deps
-    make init
```

Comment:
Why move this line?

Reply:
We need to install dependencies before we install apisix with rpm. So when we install dependencies, `make init` would fail since there is no `bin/apisix` yet.